roles/container_runtime/defaults/main.yml

```diff
@@ -145,3 +145,7 @@ l_docker_image_default: "{{ l_docker_image_prepend }}:{{ l_docker_image_tag }}"
 l_docker_image: "{{ openshift_docker_systemcontainer_image_override | default(l_docker_image_default) }}"
 
 l_is_node_system_container: "{{ (openshift_use_node_system_container | default(openshift_use_system_containers | default(false)) | bool) }}"
+
+l_crio_openshift_version: "{{ openshift_image_tag | default(openshift_release) | default('latest') | string }}"
```

Comment:
Looks like we can remove l_crio_openshift_version.  This logic has already been implemented in this vars file by 'l_openshift_image_tag'